examples/spl_composites/com.ibm.streamsx.testing.java/src/com/ibm/streamsx/testing/java/TestCustom.java

```diff
@@ -0,0 +1,85 @@
+package com.ibm.streamsx.testing.java;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.ibm.streams.operator.StreamSchema;
+import com.ibm.streams.operator.Type;
+import com.ibm.streamsx.topology.TStream;
+import com.ibm.streamsx.topology.Topology;
+import com.ibm.streamsx.topology.context.StreamsContextFactory;
+import com.ibm.streamsx.topology.spl.SPL;
+import com.ibm.streamsx.topology.spl.SPLStream;
+import com.ibm.streamsx.topology.tester.Condition;
+import com.ibm.streamsx.topology.tester.Tester;
+
+
+/*
+ * This class hows how one can test a SPL composite operator using JUnit
+ * and the Streams Java Application API.
+ */
+public class TestCustom {
```

Comment:
should lead by example on naming, this is testing the `MultipyBy` operator, not 'Custom', so rename to `TestMultipyBy`.

Reply:
Done